Dockerfile

```diff
@@ -54,8 +54,8 @@ RUN CGO_ENABLED=1 GOOS=linux go build \
 
 FROM ubuntu:20.04 as artifact
 
-RUN apt update
-RUN apt install -y git autoconf libtool g++ make
+RUN apt-get update
```

Comment:
Not really a problem, but is that change needed? `apt` should be available on all distros.

Reply:
```console
% cat Dockerfile 
FROM ubuntu:20.04
RUN apt update
% hadolint Dockerfile 
Dockerfile:2 DL3027 warning: Do not use apt as it is meant to be a end-user tool, use apt-get or apt-cache instead
% docker build -t test .                     
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM ubuntu:20.04
 ---> 597ce1600cf4
Step 2/2 : RUN apt update
 ---> Running in f393fb47d390

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

[...]
%
```